docs/nav-teams/BeebleBum.md

```diff
@@ -1,4 +1,277 @@
+    - While moving along the track, use an ease-in and ease-out motion by providing non-constant power from the motors. After each reverse in direction, generate a random number corresponding to a random location on the track. Assign weightage to such randomized location and use it as a parameter to determine the next reverse location or the next counter-attack location based the detected enemy location. If the sentry is hit while stopped at a certain location, immediately escape to a random location while spinning the armor layer. If the enemy is being attacked by the sentry at the same time, shoot the enemy while navigating to the new location
+
+### Manpower Arrangement
+
+|No|Role       |Description                          |Assigned to  |
+|--|-----------|-------------------------------------|-------------|
+|1 |Mechanical |Chassis; Suspension                  |Yiming       |
+|2 |Mechanical |Gun Control; Gimbal System           |Open Position|
+|3 |Electrical |Power management; Circuit Design; Fail-safe|Tiedong|
+|4 |Electronics|Sensor interface; Close-loop Control |Ruochen      |
+|5 |Software   |Linux OS; Sensor Drivers; UI         |Open Position|
+|6 |Software   |Computer Vision                      |Bozhao       |
+|7 |Software   |Motion Planning                      |Zesong       |
+|8 |Operation  |Chinese-English Translation, Publicity|Yiming      |
+|9 |Operation  |Accounting                           |Open Position|
+
+For interested people to join our team, please email <a href = "mailto: liruochen@u.nus.edu">liruochen@u.nus.edu</a>
+
+### Hardware List and Budget
+
+- Hardware List:
+
+![Chassis](./assets/team-beeblebum/sentry/hwlist_chassis.png)
+![Weapon](./assets/team-beeblebum/sentry/hwlist_weapon.png)
+![AI](./assets/team-beeblebum/sentry/hwlist_ai.png)
+
+## Project Timeline
+
+![Timeline](./assets/team-beeblebum/timeline.png)
+
+## Training Timeline
+
+> The team need to make good use of the open source materials to familiarize with their respective technical skills.
+> Each team can divide members into subgroups to conduct group learning and regular group discussion. 
+> The proposed training timeline is shown below.
+
+![Training Timeline](./assets/team-beeblebum/training_timeline.png)
+
+## Content Sharing
+
+> As first-year participants, it is important to ensure efficient communication and sharing of information between and within the team.
+> In order to ensure efficient online collaboration, here are proposed platform and sources of information.
+
+![Sharing Platforms](./assets/team-beeblebum/sharing_platforms.png)
+
+<style>
+details
+{
+  cursor: pointer;
+}
+img
+{
+  -moz-transition: all 0.5s ease;
+  -webkit-transition: all 0.5s ease;
+  -o-transition: all 0.5s ease;
+  transition: all 0.5s ease;
+}
+img:hover
+{
+  -moz-transform: scale(1.1);
+  -webkit-transform: scale(1.1);
+  -o-transform: scale(1.1);
+  transform: scale(1.1);
+}
+</style>
```

Comment:
The CSS code is loaded to all pages in the generated website, affecting the behaviours of other pages including the logo on the homepage.

Reply:
Possible solution:
Select based on class instead of element